e2e_tests/conftest.py

```diff
@@ -0,0 +1,199 @@
+import threading
+from wsgiref.simple_server import make_server
+
+import optuna
+from optuna_dashboard import wsgi
+import pytest
+
+
+@pytest.fixture(scope="session")
+def port() -> int:
+    return 8081
```

Comment:
How about using more larger address number to avoid the port conflict.
 
```suggestion
    port = 38080
```

Or we can find a free port like below.
https://www.programcreek.com/python/?CodeExample=get+free+port

Reply:
Fixed.